tools/transforms/shared/utils/markdown-utils.ts

```diff
@@ -28,6 +28,7 @@ export function replaceFilename(
   const filename = text.slice(startIndex + 1, endIndex);
   return (
     `
+    <app-copy-button>
```

Comment:
Should we show the copy button for code snippets that don't have the filename specified?

Example:

![image](https://github.com/user-attachments/assets/58f0add8-d04c-4aba-9d0d-343699ecd21b)

I would think so

Reply:
I agree, copy button for code snippets without filenames would improve usability. Let’s go ahead with this change!